packages/office-addin-usage-data-web/test/test.ts

```diff
@@ -0,0 +1,252 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
```

Comment:
do these tests all pass? Seems like they aren't all relevant to this package

Reply:
They all pass on my machine, but there probably are redundancies, since it was just a copy/paste from the non-web version and a change to use the web methods. I'll see what makes sense to remove.